src/Http/Http.Abstractions/src/IAllowAnonymous.cs

```diff
@@ -4,7 +4,7 @@
 namespace Microsoft.AspNetCore.Authorization
 {
     /// <summary>
-    /// Marker interface to enable the <see cref="AllowAnonymousAttribute"/>.
+    /// Marker interface to allow anonymous.
```

Comment:
This doc comment makes me sad.

Reply:
Do you have suggestions on what we could do to make this better?

Reply:
```suggestion
    /// Marker interface to allow access to anonymous users.
```

maybe?